cosmo_tester/test_suites/test_snapshots_from_321_to_33/hello_world_old_new_manager_test.py

```diff
@@ -0,0 +1,555 @@
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+import os
+import jinja2
+import pkg_resources
+import subprocess
+import shutil
+
+import cosmo_tester
+from cosmo_tester.framework.testenv import TestCase
+from cosmo_tester.framework.git_helper import clone
+from cosmo_tester.framework.util import create_rest_client
+
+import requests.packages.urllib3
+requests.packages.urllib3.disable_warnings()
+
+VIRTUALENV_NAME = 'env'
+HELLOWORLD_APP_NAME = 'helloworld'
+SNAPSHOT_NAME = 'snap'
+UPDATED_BLUEPRINT_NAME = 'openstack-manager-blueprint.yaml'
+MANAGER_KEY_FILE_NAME = 'manager_kp.pem'
+AGENTS_KEY_FILE_NAME = 'agents_kp.pem'
+RUNTIME_PROPERTY_NAME = 'rpvalue'
+
+PREPARE_TEMPLATE_NAME = 'prepare_cli_3_2_1.template'
+PREPARE_SCRIPT_NAME = 'prepare_old_cli.sh'
+
+HELLOWORLD_TEMPLATE_NAME = 'helloworld_inputs.template'
+HELLOWORLD_INPUTS_NAME = 'helloworld_inputs.yaml'
+
+INSTALL_TEMPLATE_NAME = 'install_helloworld_3_2_1.template'
+INSTALL_SCRIPT_NAME = 'install_helloworld.sh'
+
+SNAPSHOT_TEMPLATE_NAME = 'create_and_download_snapshot_3_2_1.template'
+SNAPSHOT_SCRIPT_NAME = 'create_and_download_snapshot.sh'
+
+RUN_PYTHON_TEMPLATE_NAME = 'update_user_and_credentials_3_2_1.template'
+RUN_PYTHON_SCRIPT_NAME = 'update_user_and_credentials_3_2_1.sh'
+
+CLEAR_TEMPLATE_NAME = 'clear_manager_3_2_1.template'
+CLEAR_SCRIPT_NAME = 'clear_manager_3_2_1.sh'
+
+BOOTSTRAP_TEMPLATE_NAME = 'bootstrap_manager_3_2_1.template'
+BOOTSTRAP_SCRIPT_NAME = 'bootstrap_manager_3_2_1.sh'
+
+TEARDOWN_TEMPLATE_NAME = 'teardown_manager_3_2_1.template'
+TEARDOWN_SCRIPT_NAME = 'teardown_manager_3_2_1.sh'
+
+OLD_MANAGER_TEMPLATE_NAME = 'manager_3_2_1_inputs.template'
+OLD_MANAGER_INPUTS_NAME = 'manager_3_2_1_inputs.yaml'
+
+NEW_MANAGER_TEMPLATE_NAME = 'manager_3_3_inputs.template'
+NEW_MANAGER_INPUTS_NAME = 'manager_3_3_inputs.yaml'
+
+UPDATE_SCRIPT_NAME = 'update_user_and_credentials_3_2_1.py'
+
+
+class HelloWorldOldNewManagerTest(TestCase):
```

Comment:
Maybe HelloWorldSnapshotMigrationFrom3_2_1to3_3Test
or
Maybe HelloWorldSnapshotMigrationFrom_3_2_1_to_3_3_Test

I know.. it's hard to come up with a good name here :/